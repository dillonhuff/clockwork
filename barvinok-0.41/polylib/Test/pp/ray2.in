1 5
1 1 1 -1 0
0 3
a
