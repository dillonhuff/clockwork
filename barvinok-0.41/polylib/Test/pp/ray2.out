---------------------------------------
Overall rays :
Line:   [   1   -1  ]
Ray:    [   0    1  ]
---------------------------------------
Domain :
          1 >= 0

Vertices :
[ a,  0 ]
